docs/how-to/uc-image-creation.rst

```diff
@@ -0,0 +1,214 @@
+How to create a real-time Ubuntu Core image
+===========================================
+
+.. tip::
+
+    This guide assumes you have access to features available exclusively to `IoT App Store`_ users.
```

Comment:
The term of art now is "Dedicated Snap Store".

Reply:
There are several places I'm not going to point out where "IoT App Store" is used to avoid noise.
I noticed that some of the links used reference "IoT App Store", I've asked Store folx to confirm if this name is accurate (as the name should have changed in May).

Reply:
I've asked Holly about the state of this as it's currently in use [in the overview](https://ubuntu.com/core/docs/store-overview), I'll let you know.

Reply:
The core documentation that I was referring to aren't maintained. I had discovered them via search engines. 
Changed: https://github.com/canonical/real-time-ubuntu-docs/pull/51/commits/2a7c839bc2c2f867446b51097e1e2512f18e8ef3